<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditInputNeuronTab</class>
 <widget class="QewExtensibleDialog" name="EditInputNeuronTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>162</height>
   </rect>
  </property>
  <property name="caption" stdset="0">
   <string>Input generator parameters</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Input generator</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Identification</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="identSpinBox">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="QLabel" name="labelConnection">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Disconnected</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Group identification</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="groupComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Sonar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Laser</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Infrared</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bumpers</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>322</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 
 <customwidgets>
  <customwidget>
   <class>QewExtensibleDialog</class>
   <extends>QDialog</extends>
   <header>qewextensibledialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">/usr/include/qew5/qewextensibledialog.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
